<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1962</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(36, 36, 36);</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">
color: rgb(36, 36, 36);

</string>
      </property>
      <property name="text">
       <string>asdasdasdasdasd</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-right-color: rgb(24,75, 84);
font: 75 italic 38pt &quot;Century Schoolbook L&quot;;


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>SWIFT TYPER_</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">
color: rgb(36, 36, 36);

</string>
        </property>
        <property name="text">
         <string>asdasdasdasdasd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">color: rgb(36, 36, 36);

</string>
        </property>
        <property name="text">
         <string>asdasdasdasdasd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="input_para"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
font: 75 22pt &quot;TlwgTypewriter&quot;;
background-color: rgb(15, 15,15);
</string>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLCDNumber" name="lcd_wpm">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>10000000</width>
        <height>10000000</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
font: 75 22pt &quot;TlwgTypewriter&quot;;
background-color: rgb(15, 15,15);
</string>
      </property>
      <property name="text">
       <string>Accuracy:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLCDNumber" name="lcd_accuracy">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
font: 75 22pt &quot;TlwgTypewriter&quot;;
background-color: rgb(15, 15,15);
</string>
      </property>
      <property name="text">
       <string>Time left:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLCDNumber" name="lcd_time">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="7" colspan="2">
     <widget class="QTextEdit" name="output_para">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(24, 75, 84);


font: 63 20pt &quot;Ubuntu&quot;;
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" rowspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ht1.qrc">:/ht/credenz2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>input_para</zorder>
   <zorder>label_8</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>lcd_wpm</zorder>
   <zorder>label_2</zorder>
   <zorder>lcd_accuracy</zorder>
   <zorder>label_3</zorder>
   <zorder>lcd_time</zorder>
   <zorder>output_para</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1962</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ht1.qrc"/>
  <include location="ht1.qrc"/>
 </resources>
 <connections/>
</ui>
